NetMQ.ReactiveExtensions/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {    
+    "NetMQ": "4.0.0-rc5",
+    "protobuf-net": "2.1.0",
+    "System.Reactive.Core": "3.1.0"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.1"
+        }
+      },
+      "imports": "dnxcore50"
+    },
+    "netstandard1.6": {
+      "dependencies": {
+        "NETStandard.Library": "1.6.0"
+      }
+    },
+    "net45": {
+      "dependencies": {
+        "System.Xml.Linq" : "*" 
```

Comment:
Just tried this on my machine and it doesn't seem necessary. Rationale here is that the full .NET framework includes the `System.Xml.Linq` namespace. I think in .NET Core it's not part of the core framework, but is instead available as a package from NuGet. So you tend to see familiar namespaces listed as `dependencies` under `netstandard` and `netcoreapp` but not `net35`, `net45` as they are bundled. It took me a while to get the distinction here.